test/precision_test.exs

```diff
@@ -0,0 +1,31 @@
+defmodule PrecisionTest do
+  use ExUnit.Case
```

Comment:
I think you should set `async: false` here to prevent other tests randomly getting `precision: :microsecond`